Ekzitus

Ekzitus opened the pull request.

Commit:
Mission complete

zakharkin-ivan commented:
Добрый день.
У меня есть несколько вопросов и замечаний по выполненному заданию:
1. Можно ли избавиться от бесконечного кол-ва async-await в Update?
2. Прошу сделать SelectAsync проще, без множества локальных переменных
3. Все запросы следует вынести в константы.
4. В чём минус использования интерполированной или параметризованной строки в запросе?

Commit:
1. Исправил бесконечное кол-ва async-await в Update
2. Сделал SelectAsync без локальных переменных
3. Вынес запросы в константы
4. Использование интерполированной или параметризованной строки в запросе опасно попадением в запрос постароннего кода
